deep-learning/_posts/2018-11-09-Incremental-Learning.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: review
+title:  "Incremental Learning for Semantic Segmentation of Large-Scale Remote Sensing Data"
+tags:   Incremental learning, catastrophic forgetting, deep-learning, segmentation, remote-sensing,
+author: Charles Authier
+pdf:  https://arxiv.org/abs/1810.12448
+cite:
+  authors: "Onur Tasar, Yuliya Tarabalka, Pierre Alliez"
+  title:   "Incremental Learning for Semantic Segmentation of Large-Scale Remote Sensing Data"
+  venue:   "Arxiv, 29 oct 2018"
+---
+
+Code: https://project.inria.fr/epitome/inc_learn/
+
+## Method
+One big probleme, catastrophic forgetting: a significant performance drop for the already learned classes when new classes are added on the data, having no annotations for the old classes.
+They propose an incremental learning methodology, enabling to learn segmenting new classes without hindering dense labeling abilities for the previous classes, although the entire previous data are not accessible.
+
+The key points of the proposed approach are adapting the network to learn new as well as old classes on the new training data, and allowing it to remember the previously learned information for the old classes.
+
+For adaptation, they keep a frozen copy of the previously trained network, which is used as a memory for the updated network in absence of annotations for the former classes.
+The updated network minimizes a loss function, which balances the discrepancy between outputs for the previous classes from the memory and updated networks, and the mis-classification rate between outputs for the new classes from the updated network and the new ground-truth.
+
+![](/deep-learning/images/incrementallearning/intro.png)
+
+## Loss
+
+For a total of three losses:
+- $$L_{class}$$ (sigmoid cross entropy loss)
+
+![](/deep-learning/images/incrementallearning/loss1.png)
+
+- $$L_{distill}$$, measures the disparity between $$y_{mem}$$ samples from the memory network, and $$y_{up_prev}$$ from the updated network.
+
+![](/deep-learning/images/incrementallearning/loss2.png)
+
+- $$L_{adapt}$$ is the Loss use during the adaptation
+
+![](/deep-learning/images/incrementallearning/loss12.png)
+
+- $$L_{rem}$$ is the costum loss for the Incremental learning
```

Comment:
is the costum loss for the Incremental learning -> is the custom loss for the incremental learning

Reply:
Has not been yet changed. Am I misunderstanding the meaning of the term?